.travis.yml

```diff
@@ -1,24 +1,28 @@
+---
 language: node_js
-python: 2.7
+python:
+  - "2.7"
 install:
-- npm install
+  - npm install
+  - pip install --user tox
 script:
-- npm test
-- python setup.py bdist_wheel
+  - npm test
+  - tox
+  - python setup.py bdist_wheel
```

Comment:
Forward looking: use `bdist_wheel --universal` instead [to support Python 3 compatibility](https://wheel.readthedocs.io/en/stable/#defining-the-python-version)?